src/style.css

```
@@ -0,0 +1,31 @@
+.content {
+  display: flex;
+  flex-direction: column;
+  width: 106px;
+  justify-content: space-between;
+}
+
+.item {
```

Reset browser's default margin

```suggestion
body {
  margin: 0;
}

.item {
```